docs/api/sdk/README.md

```diff
@@ -402,33 +402,44 @@ Use the `exception` type for errors and crashes.
 
 Each exception object contains further fields.
 
-| Field     | Type   | Optional | Comment                     |
-| --------- | ------ | -------- | --------------------------- |
-| `type`    | string | No       | Type of the exception       |
-| `message` | string | No       | Error message text          |
-| `frames`  | array  | Yes      | Array of stackframe objects |
+| Field                      | Type    | Optional | Comment                        |
+| -------------------------- | ------- | -------- | ------------------------------ |
+| `type`                     | string  | No       | Type of the exception          |
+| `message`                  | string  | No       | Error message text             |
+| `frames`                   | array   | Yes      | Array of stackframe objects    |
+| `signal`                   | string  | Yes      | BSD termination signal         |
+| `thread-name`              | string  | Yes      | Name of thread                 |
+| `thread-sequence`          | Int     | Yes      | Thread number                  |
+| `operatingSystemBuild`     | string  | Yes      | Operating system build number  |
+| `cpuArch`                  | string  | Yes      | The CPU architecture           |
 
 `thread` objects
 
 Each thread object contains further fields.
 
-| Field    | Type   | Optional | Comment                     |
-| -------- | ------ | -------- | --------------------------- |
-| `name`   | string | Yes      | Name of thread              |
-| `frames` | array  | Yes      | Array of stackframe objects |
+| Field      | Type   | Optional | Comment                     |
+| ---------- | ------ | -------- | --------------------------- |
+| `name`     | string | Yes      | Name of thread              |
+| `frames`   | array  | Yes      | Array of stackframe objects |
+| `sequence` | Int    | Yes      | Thread number               |
```

Comment:
Let's describe what `sequence` does in more detail as it's not very intuitive by itself.

Reply:
update the `exception` object to mention how the thread sequences are used.